.github/workflows/flowzone.yml

```diff
@@ -0,0 +1,33 @@
+name: Flowzone
+
+on:
+  pull_request:
+    types: [opened, synchronize, closed]
+    branches:
+      - "main"
+      - "master"
+
+jobs:
+  flowzone:
+    name: Flowzone Linux
```

Comment:
Using anything other than the expected `Flowzone` name invites weirdness, especially since the `runs_on` below specifies that this can also run on things other than linux